Receivers/unix/pulseaudio.c

```diff
@@ -161,7 +163,7 @@ int pulse_output_send(receiver_data_t *data)
         "Scream",
```

Comment:
It works, I can see the correct names in pavucontrol but at least on KDE all I see is the application name Scream. See:

![image](https://user-images.githubusercontent.com/6640057/78812211-59321d80-79cb-11ea-81b2-ecb8e2c8382a.png)

I have no idea how it works on other systems but if it's common to report only the application name maybe we can set it to "Scream (stream name)" ?

Reply:
If you rightclick the kde volume control and access the Audio setting page, it shows the stream descriptions like pavucontrol does.

<img width="488" alt="image" src="https://user-images.githubusercontent.com/341818/78869434-e5ccf200-7a44-11ea-9b32-6fcbc8d09f3b.png">

I agree it would be nicer if it would be visible on the quick-access volume applet too, but none of the other applications seem to add anything to the application name itself.

Reply:
Yeah, you are right. I see it's common this way.